deploy/crds/chaosengine.yaml

```diff
@@ -0,0 +1,101 @@
+---
+## This is the chaos engine resource requested by dev for the nginx 
+## app, i.e., the user facing custom resource. Mapped to a dedicated
+## controller which triggers the actual chaos experiments as per 
+## defined schedule. The app developer is expected to annotate the
+## application with litmuschaos.io/engine: <engine-name>
+
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  ## Eventually launched chaos (cron) jobs will bear <engine>-<exp></reference>-<hash>
+  name: engine-nginx
+  namespace: litmus 
+spec:
+
+  ## This will be used to identify AUT (Application-Under-Test) 
+  ## This app will be value of probed for annotation "litmuschaos.io/chaos"
+  ## to check whether it can be subjected to chaos. 
+
+  appinfo: 
+    appns: default
+    applabel: "app=nginx"
+  
+  ## The chaos experiments themselves will be "pulled" via
+  ## downloadable litmus charts and versioned against it
+  ## The chaos operator comes with a default set of experiments
+
+  experiments:
+    - name: pod-delete 
+      spec:
+
+        # The experiments can be ranked on execution priority
+        rank: 1
+
+        # An experiment may need additional data points
+        # "apart" from the "chaos-specific params" defined
+        # in the template. See spec of flaky-networks for
+        # case-in-point
+
+        # The same will be available in experiment spec as
+        # well, this definition here will override that.
+ 
+        components: 
+         
+    - name: network-delay
+      spec:
+        rank: 2 
+        component:
+          nwinterface: ""
+
+    - name: disk-fill
+      spec: 
+        rank: 4
+        component:
+          disk: ""
+        
+        # Individual experiments can also have schedule
+
+        schedule: 
+          interval: ""
+          excluded-times: ""
```

Comment:
should all the property names follow a certain naming standard?
e.g. why not `excludedTimes` & `excludedDays`